cinch/roles/nginx/tasks/nginx.yml

```diff
@@ -0,0 +1,67 @@
+- name: install nginx
+  yum:
```

Comment:
People use "package" instead of "yum" as it will continue our support for Fedora.